packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable dot-notation */
 /* eslint-disable @typescript-eslint/no-shadow */
-import * as MediaSDK from '@webex/internal-media-core';
+import * as InternalMediaCoreModule from '@webex/internal-media-core';
```

Comment:
@Kesari3008 I wouldn't say that it's a must, but in one file, there were two imports with different names for `'@webex/internal-media-core'` in one file. I've changed how we mock `'@webex/internal-media-core'` for all test cases. We can discuss if you want to use another variable for this somewhere, but it feels right to align the naming if we want to make our lives easier with the codebase. This can be done separately if you feel it could be an issue. But so far, it's just a naming without logic change part, so it should be okay.